web/src/OverviewTableStatus.tsx

```diff
@@ -98,20 +98,28 @@ export default function OverviewTableStatus(props: OverviewTableStatusProps) {
       msg = isBuild ? "Update Pending" : "Runtime Pending"
       classes = "is-pending"
       break
-    case ResourceStatus.Warning:
+
+    case ResourceStatus.Warning: {
+      let buildDurText = lastBuildDur
+        ? ` in ${formatBuildDuration(lastBuildDur)}`
+        : ""
+      icon = <WarningSvg fill={Color.yellow} width="10px" height="10px" />
```

Comment:
i noticed that the icon shrinks a ton when the table shrinks. adding `flex-shrink: 0;` to the `StatusIcon` class fixed it!

Reply:
done